src/Container/AddWmsPanel/AddWmsPanel.jsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from 'antd';
+
+import { Panel, SimpleButton, Titlebar } from '../../index';
+
+import './AddWmsPanel.less';
+import AddWmsLayerEntry from './AddWmsLayerEntry/AddWmsLayerEntry.jsx';
+
+/**
+ * Panel containing a (checkable) list of AddWmsLayerEntry instances.
+ * This class can be used e.g with a result obtained by ol WMS capabilities
+ * parser, in particular objects in `Capability.Layer.Layer`
+ *
+ * @class The AddWmsPanel
+ * @extends React.Component
+ */
+export class AddWmsPanel extends React.Component {
+
+  /**
+   * The prop types.
+   * @type {Object}
+   */
+  static propTypes = {
+    /**
+     * Array containing layers (e.g. `Capability.Layer.Layer` of ol capabilities
+     * parser)
+     * @type {Array} -- required
+     */
+    wmsLayers: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
I think it would be better to work with instances of `OlLayer` here. The parser could be in the `CapabilitiesUtil` described above. I don't like the idea of working with diffrent "layerlike" objects across the framework.
You could then implement an "real" `addToMap` method wich would call the property method `onLayerAddToMap` after add. Ofcourse the map should be a required prop.

Reply:
I replaced all "custom" objects representing a layer by an instance of `OlLayerTile`.

Reply:
See c2889cb